### PR TITLE
feat(client): hide the outline instead of removing it

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

In high contrast mode, which used for accessibility, outline is needed to provide contrast, this aims to show it in high contrast mode

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: client/src/components/Header/components/universal-nav.css
Comment (bbsmooth) [inline client/src/components/Header/components/universal-nav.css:179]:
```suggestion
```

Revert this back to `outline: none`. This rule set is for when the button has focus but it was caused by something other than the keyboard (e.g. mouse, touch). In that case we don't want a focus outline to show.

File: client/src/components/Header/components/universal-nav.css
Comment (bbsmooth) [inline client/src/components/Header/components/universal-nav.css:351]:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/components/Header/components/universal-nav.css
Comment (bbsmooth) [inline client/src/components/Header/components/universal-nav.css:495]:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/templates/Challenges/classic/classic.css
Comment (bbsmooth) [inline client/src/templates/Challenges/classic/classic.css:145]:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/templates/Challenges/components/challenge-title.css
Comment (bbsmooth) [inline client/src/templates/Challenges/components/challenge-title.css:44]:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/templates/Challenges/components/output.css
Comment (bbsmooth) [inline client/src/templates/Challenges/components/output.css:23]:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/templates/Introduction/intro.css
Comment (bbsmooth) [inline client/src/templates/Introduction/intro.css:61]:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.

File: client/src/templates/Introduction/intro.css
Comment (bbsmooth) [inline client/src/templates/Introduction/intro.css:269]:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.

**Comment by bbsmooth** [comment]
We are going to be able to get rid of most of this outline related CSS once PR #49579 is merged. So it might be best to hold off on this until that happens.

**Comment by Sboonny** [comment]
When you have free time, can you take another look?

**Comment by bbsmooth** [comment]
@Sboonny, I can't seem to run this. I'm getting the following:

Your pnpm version is incompatible with "/home/bbsmooth/freecodecamp/freeCodeCamp".

Expected version: 7
Got: 8.1.0

**Comment by raisedadead** [comment]
@bbsmooth you should be able to test this now. I just did a quick rebase for this PR.

**Review by Sembauke — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**
